### ISSUE TITLE
FR: Don't dim current/active tab when it loses focus

### ISSUE THREAD

**Issue by churnish** [smikula/obsidian-limelight#14]
Currently, if moving focus away from any open notes (e.g. onto the file explorer), the plugin will dim all tabs, including the current/active tab. Ideally, the current/active tab would always stay undimmed. This would be useful, for example, to know in which pane a note will be opened when selecting note in the file explorer.

**Comment by smikula** [comment]
Good idea.  I'm not actively working on this project, but when I come back to it I'll look into this.
